.github/workflows/draft-new-release.yml

```diff
@@ -0,0 +1,77 @@
+name: "Draft new release"
+
+on:
+  issues:
+    types: [opened, labeled]
+
+jobs:
+  draft-new-release:
+    name: "Draft a new release"
+    runs-on: ubuntu-latest
+    # Only run for issues with a specific title and label. Not strictly required but makes finding the release issue again later easier.
+    # There is also a whitelist that you may want to use to restrict, who can trigger this workflow.
+    # Unfortunately, we cannot create an array on the fly, so the whitelist is just comma-separated.
+    if: startsWith(github.event.issue.title, 'Release version') && contains(github.event.issue.labels.*.name, 'release') && contains('thomaseizinger,bonomat,D4nte,da-kami,luckysori,tcharding,rishflab', github.event.issue.user.login)
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Extract version from issue title
+        run: |
+          TITLE="${{ github.event.issue.title }}"
+          VERSION=${TITLE#Release version }
+
+          echo "::set-env name=RELEASE_VERSION::$VERSION"
+
+      - name: Create release branch
+        run: git checkout -b release/${{ env.RELEASE_VERSION }}
+
+      - name: Update changelog
+        uses: thomaseizinger/keep-a-changelog-new-release@1.1.0
+        with:
+          version: ${{ env.RELEASE_VERSION }}
+
+      - name: Initialize mandatory git config
+        run: |
+          git config user.name "GitHub actions"
+          git config user.email noreply@github.com
+
+      - name: Bump version in Cargo.toml
+        uses: thomaseizinger/set-crate-version@1.0.0
+        with:
+          version: ${{ env.RELEASE_VERSION }}
+          manifest: cnd/Cargo.toml
+
+      - name: Update Cargo.lock
+        uses: actions-rs/cargo@v1
+        with:
+          command: update
+          args: --package cnd
```

Comment:
Why updating only for `cnd`?

Reply:
Only the version of the cnd crate changed, why should I update anything else? As you mentioned, doing an overall update would just cause merge conflicts.